code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -644,6 +648,21 @@
 					M.adjustBruteLoss(rand(0,15))
 					M.adjustToxLoss(rand(0,15))
 					M.adjustFireLoss(rand(0,15))
+					if(ishuman(M) && M.mind && prob(10)) // To prevent people from using this on monkies to cause pandemics
+						var/mob/living/carbon/human/H = M
+						// Their body is being suffused with magical life-inducing stuff - so other things get lively, too
+						// Things are going to get interesting for the reviving party, too
+						var/virus_type = pick(list(
+							/datum/disease/advance/flu,
+							/datum/disease/advance/cold,
+							/datum/disease/brainrot,
+							/datum/disease/magnitis,
+							/datum/disease/fake_gbs, // honk
+							/datum/disease/cold9
+						))
+						var/datum/disease/D = new virus_type()
+						H.AddDisease(D)
```

Comment:
(defib time is 3 not 5 minutes)

"Individual tuning levers are better than group levers" - 
I wanna experiment with this but eh alright I'll change it to something else

Instead, I'll change it to making it revive only some of the person's non-vital limbs/organs, leaving the other ones in a state of requiring surgical attention/amputation.